code-scanning/frogbot-scan-pr.yml

```diff
@@ -0,0 +1,112 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: "Frogbot Scan"
+on:
+  pull_request_target:
+    types: [ opened, synchronize ]
+permissions:
+  pull-requests: write
+jobs:
+  scan-pull-request:
+    runs-on: ubuntu-latest
+    # A pull request needs to be approved, before Frogbot scans it. Any GitHub user who is associated with the
+    # "frogbot" GitHub environment can approve the pull request to be scanned.
+    environment: frogbot
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      # Install prerequisites - uncomment the relevant ones
+
+      # - uses: actions/setup-go@v3
+      #   with:
+      #     go-version: 1.17.x
+
+      # - uses: actions/setup-java@v3
+      #    with:
+      #      java-version: "11"
+      #      distribution: "temurin"
+
+      # - uses: actions/setup-node@v3
+      #    with:
+      #      node-version: "16.x"
+
+      # - uses: gradle/gradle-build-action@v2
+
+      # - uses: actions/setup-dotnet@v2
+      #  with:
+      #    dotnet-version: "3.1.x"
+
+      # - uses: nuget/setup-nuget@v1
+      #  with:
+      #    nuget-version: "5.x"
+
+      # - uses: actions/setup-python@v3
+      #  with:
+      #    python-version: "3.x"
+
+      # - run: pipx install pipenv
+
+      - uses: jfrog/frogbot@v2
```

Comment:
Kindly use commit sha instead of release tags

Reply:
Done :)